game.js

```diff
@@ -1,3 +1,48 @@
-// Write all your code here
-// Replace the contents of this file with your own code
-console.log('Hello, world!');
+
+
+
+let choices = ["rock", "paper", "scissors"];
+
+function computerPlay(){
+	return choices[Math.floor(Math.random()*3)] ;
+}
+
+function playRound(playerSelection,computerSelection){
+	
+	if( playerSelection !== "rock" && playerSelection !== "paper" && playerSelection !== "scissors" ){
```

Comment:
Hello Hamza
I think if you convert the playerSelection to lower case at the beginning of the function it will let the player to insert the choice as he/she want , for example : rock, Rock, ROCK.
```suggestion
        playerSelection = playerSelection..toLowerCase();
	if( playerSelection !== "rock" && playerSelection !== "paper" && playerSelection !== "scissors" ){
```